package.json

```diff
@@ -14,7 +14,7 @@
     "start:debug": "nest start --debug --watch",
     "start:prod": "node dist/main",
     "lint": "eslint \"{src,apps,libs,test}/**/*.ts\" --fix",
-    "test": "cross-env NODE_ENV=automated_tests jest",
+    "test": "cross-env NODE_ENV=automated_tests jest --testTimeout=20000",
```

Comment:
Revert this line. Check first if you can wait in the specific test instead of adding the timeout globally.